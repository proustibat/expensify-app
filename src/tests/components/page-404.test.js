import React from 'react';
import { shallow } from 'enzyme';
import Page404 from "../../components/page-404";

test( 'should render 404 page correctly', () => {
    const wrapper = shallow( <Page404 /> );
    expect( wrapper ).toMatchSnapshot();
} );
