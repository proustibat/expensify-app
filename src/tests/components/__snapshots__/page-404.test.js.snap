// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 404 page correctly 1`] = `
<div>
  404 - 
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
